server/raft.go

```diff
@@ -3761,6 +3761,10 @@ func (n *raft) readTermVote() (term uint64, voted string, err error) {
 	if err != nil {
 		return 0, noVote, err
 	}
+	if len(buf) < 8 {
```

Comment:
I wonder, should we return an error here?

Though I see that we don't handle errors from `readTermVote()` anyway, beyond checking if its nil or not.

Reply:
An error isn't really actionable in that scenario anyway as we'd still end up resetting back to term 0, in which case we'd might as well just return term 0.